starwars/src/App.js

```diff
@@ -31,8 +32,14 @@ class App extends Component {
 
   render() {
     return (
+
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        <List  height={this.state.starwarsChars}/>
```

Comment:
You can probably remove this component completely and just work with your CharButton component.